### DISCUSSION TITLE
azure-native.kubernetes.ConnectedCluster what is it?

### DISCUSSION THREAD

**Discussion by KIRY4** [pulumi/pulumi#14074] [Pulumi Packages] [answered]
Hello everyone!

Is anybody can share some link on Azure or any other docs where I can read what is Azure Kubernetes **Connected Cluster?** Connected to what? )) I just want to test provisioning simple AKS cluster but found 2nd option in documentation called Connection Cluster. Is it somehow related to Azure Arc or private endpoint or private link?

Thanks!

**Comment by sam-cogan** [comment, marked as answer]
A connected cluster is an on-prem (or other cloud) cluster connected via Azure Arc.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: KIRY4 ***@***.***>
Sent: Saturday, September 30, 2023 11:13:06 AM
To: pulumi/pulumi ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [pulumi/pulumi] azure-native.kubernetes.ConnectedCluster what is it? (Discussion #14074)


Hello everyone!

Is anybody can share some link on Azure or any other docs where I can read what is Azure Kubernetes Connected Cluster? Connected to what? )) I just want to test provisioning simple AKS cluster but found 2nd option in documentation called Connection Cluster. Is it somehow related to Azure Arc or private endpoint or private link?

Thanks!

**Comment by KIRY4** [comment]
@sam-cogan thanks for the explanation!
